gulpfile.js

```diff
@@ -10,7 +10,8 @@ gulp.task('compile', function() {
   var result = gulp.src('src/ModalBox.ts')
       .pipe(ts({
         declaration: true,
-        out: 'bin/ModalBox.js',
+        module: 'ES2015',
```

Comment:
You've been able to test it locally that it works correctly when you require it in an external project ?

Reply:
Yes, I testing inside the interface editor project, using both `require` and `import`.